utils/aws_sigv4.cc

```diff
@@ -9,6 +9,7 @@
 #include <gnutls/crypto.h>
 #include "utils/aws_sigv4.hh"
 #include "utils/hashers.hh"
+#include "bytes.hh"
```

Comment:
Please correct me if I'm wrong, but I see only bytes_view is used, so we just need bytes_fwd.hh?

Reply:
`sstring to_hex(const bytes&)` is used in multiple places in this file. see https://github.com/scylladb/scylladb/blob/e51b2075daccf4a52c07c7b7fc903124ed0aaa82/utils/aws_sigv4.cc#L39 for an example. and this function is declared in `bytes.hh`. see https://github.com/scylladb/scylladb/blob/e51b2075daccf4a52c07c7b7fc903124ed0aaa82/bytes.hh#L42 , and this declaration does not exist in `bytes_fwd.hh`. so we have to include `bytes.hh` here.